package/Dockerfile

```diff
@@ -65,7 +65,7 @@ ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 100.0.2+up0.3.2
 
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=100.0.3+up0.3.9-rc1
```

Comment:
are we waiting on fleet tag?

Reply:
Yes we are still waiting on the fleet release, this pr was originally just to bump rke but now I think it should wait for fleet.  Ill make it as a draft until we are ready.